### PR TITLE
[WIP] Account abstraction while not mining

### PR DESCRIPTION
**Pull request by jmederosalvarado**
## Changes:
- Enables Account Abstraction RPCModule independently of whether mining is enabled or not
- Adds constructor parameter to PaymasterThrottler to allow the use of different constants depending on the _mining mode_

## Types of changes

What types of changes does your code introduce?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Other (please describe): 

## Testing
**Requires testing**

- [ ] Yes
- [ ] No

**In case you checked yes, did you write tests??**

- [ ] Yes
- [ ] No

**Comments about testing , should you have some** (optional)

### CODE REVIEW

File: src/Nethermind/Nethermind.AccountAbstraction/Source/PaymasterThrottler.cs [hunk 1]
```diff
@@ -49,19 +49,16 @@ public class PaymasterThrottler : IPaymasterThrottler
                 )
             );
 
-        public PaymasterThrottler()
-        {
-            _opsSeen = new Dictionary<Address, uint>();
-            _opsIncluded = new Dictionary<Address, uint>();
+        public PaymasterThrottler() : this(true) { }
 
-            SetupAndStartTimer();
```
Comment (kristofgazso) [inline src/Nethermind/Nethermind.AccountAbstraction/Source/PaymasterThrottler.cs:57]: why did you remove this?

File: src/Nethermind/Nethermind.AccountAbstraction/Source/PaymasterThrottler.cs [hunk 2]
```diff
@@ -49,19 +49,16 @@ public class PaymasterThrottler : IPaymasterThrottler
...
+            : this(isBundler, new Dictionary<Address, uint>(), new Dictionary<Address, uint>()) { }
 
-        public PaymasterThrottler(
-            IDictionary<Address, uint> opsSeen,
-            IDictionary<Address, uint> opsIncluded
-        )
+        public PaymasterThrottler(bool isBundler,
+                IDictionary<Address, uint> opsSeen, IDictionary<Address, uint> opsIncluded)
         {
+            MinInclusionRateDenominator = isBundler ? (uint)10 : (uint)100;
```
Comment (kristofgazso) [inline src/Nethermind/Nethermind.AccountAbstraction/Source/PaymasterThrottler.cs:60]: lets call this isMiner instead to avoid confusion with flashbots

**Review by kristofgazso — CHANGES_REQUESTED** [review]

File: src/Nethermind/Nethermind.AccountAbstraction/Source/PaymasterThrottler.cs (hunk 1 above)
Comment (jmederosalvarado) [reply src/Nethermind/Nethermind.AccountAbstraction/Source/PaymasterThrottler.cs:57]: Actually this constructor now calls the other constructor from the same class, and this step is done there

Comment (jmederosalvarado) [reply src/Nethermind/Nethermind.AccountAbstraction/Source/PaymasterThrottler.cs:57]: I think this way is better, since now there is only the constructor logic is concentrated in only one place

File: src/Nethermind/Nethermind.AccountAbstraction/Source/PaymasterThrottler.cs (hunk 2 above)
Comment (jmederosalvarado) [reply src/Nethermind/Nethermind.AccountAbstraction/Source/PaymasterThrottler.cs:60]: OK, that's fair

Comment (jmederosalvarado) [reply src/Nethermind/Nethermind.AccountAbstraction/Source/PaymasterThrottler.cs]: Done

**Review by kristofgazso — APPROVED** [review]
